### PR TITLE
DFE-1021 Adding call to get a users publications - still need to work…

### PR DESCRIPTION
**Pull request by richardpopplehive**
… out how users are connected to publications and this is dummied out

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs [hunk 1]
```diff
@@ -68,6 +68,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<INotificationsService, NotificationsService>();
             services.AddTransient<IPublishingService, PublishingService>();
             services.AddTransient<IThemeService, ThemeService>();
+            services.AddTransient<IPublicationService, PublicationService>();
```
Comment (shakes90) [inline src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs]: remove blank line.

File: src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IThemeService.cs [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 using System.Collections.Generic;
-using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
 {
     public interface IThemeService
     {
-        List<Theme> GetThemes(Guid guid);
+        List<Theme> GetByUser(Guid guid);
```
Comment (jamescutts) [inline src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IThemeService.cs]: I wonder if we should go with something slightly more descriptive like `GetUserThemes` as i suspect we will end up with a bunch more stuff in here

Comment (richardpopplehive) [reply src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IThemeService.cs]: Can do

File: src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs (hunk 1 above)
Comment (richardpopplehive) [reply src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs]: Done

**Pull request merged**